include/variables_overlay.h

```diff
@@ -11618,8 +11618,8 @@ extern UNK_TYPE4 D_80B6DB88; // D_80B6DB88
 extern OverlayRelocationSection bgSinkaiKabeOverlayInfo; // D_80B6DB90
 extern u32 bgSinkaiKabeOverlayRelocations[13]; // D_80B6DBA4
 extern u32 bgSinkaiKabeOverlayInfoOffset; // D_80B6DBDC
-extern UNK_TYPE1 D_80B6DF80; // D_80B6DF80
-extern UNK_TYPE1 D_80B6DFA0; // D_80B6DFA0
+extern UNK_TYPE1 Bg_Haka_Curtain_InitVars; // Bg_Haka_Curtain_InitVars
+extern UNK_TYPE1 sInitChain; // sInitChain
```

Comment:
We should keep this unchanged for now. This file will be deleted, but having multiple variables with the same name will cause issues.

```suggestion
extern UNK_TYPE1 D_80B6DFA0; // D_80B6DFA0
```